components/metrics/http_test.go

```diff
@@ -17,7 +17,7 @@ func TestCreateRegistryAndServeHTTP_metrics_endpoint(t *testing.T) {
 	defer cancel()
 	err := reg.Register(collectors.NewBuildInfoCollector())
 	if err != nil {
-		t.Fatal(errors.Wrap(err, "registration of prometheus build info collector failed"))
+		t.Fatal(fmt.Errorf("registration of prometheus build info collector failed: %w", err))
```

Comment:
Here the previous code is somehow bad.

And you are changing it to something better, but unfortunately still bad.

See